cloud/aws/templates/aws_oidc/alarms.tf

```diff
@@ -190,5 +190,24 @@ resource "aws_cloudwatch_metric_alarm" "maximum_used_transaction_ids_too_high" {
   statistic           = "Average"
   threshold           = var.rds_max_used_transaction_ids_high_threshold
   alarm_description   = "Nearing a possible critical transaction ID wraparound. More info [here](https://aws.amazon.com/blogs/database/implement-an-early-warning-system-for-transaction-id-wraparound-in-amazon-rds-for-postgresql/)"
-  alarm_actions       = local.rds_alarm_actions
+  alarm_actions       = local.civiform_alarm_actions
+}
+
+resource "aws_cloudwatch_metric_alarm" "memory_utilization_too_high" {
+  count               = var.ecs_create_high_memory_alarm ? 1 : 0
+  alarm_name          = "ecs-${module.ecs_cluster.aws_ecs_cluster_cluster_name}-highMemoryUtilization"
```

Comment:
**nitpick:** Thoughts on using `local.name_prefix` here?

Reply:
I have gut reservations about not explicitly dimensioning the name on the dimension criteria, but I am not familiar enough to say why that might cause problems later. I think it's a pretty easy fix so let's do it.